### DISCUSSION TITLE
Moving repositories from old github account (inaccessable due to phone number change)

### DISCUSSION THREAD

**Discussion by owenthespoon2** [community/community#146662] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I recently lost access to my phone number due to moving to another country and therefore lost access to my 2 Step verification.

I tried the AI chat bot to get access but it sadly didn't help and I ended up removing my main email address to this new account.

How can I move my old repositories to this new account?

Thanks in advance.

**Comment by mecodeatlas** [comment]
Hey @owenthespoon2!

While GitHub allows repository transfers, as outlined in the [documentation](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository), you need access to your account for the process to be completed. Since you lost access due to the 2 Step Verification issue, you can open a ticket on our [Support page](https://support.github.com/contact) so they can check the best way to help you with this.
